### PR TITLE
Feature/gh 10475 ov cy test

### LINKED ISSUE #10475: Implement Cypress test for overview page
Implement Cypress test which checks if overview page loads and renders after navigation

### PR DESCRIPTION
**Pull request by Uli-Tiger**

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts [hunk 1]
```diff
@@ -283,4 +301,48 @@ context('CPQ Configuration', () => {
         });
     });
   });
+
+  describe('Overview Page', () => {
+    it.only('should display user selections on overview page', () => {
```
Comment (MichaelBoeser) [inline projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts]: please remove only statement.

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts [hunk 2]
```diff
@@ -111,3 +111,64 @@ export function verifyNotificationBannerOnOP(numberOfIssues?: number): void {
...
+  groupName: string,
+  groupIdx: number
+): void {
+  cy.get('cx-configurator-overview-form .cx-group h2')
+    .eq(groupIdx)
+    .should('contain.text', groupName);
+}
+
+/**
+ * Verifies whether the group header displayed.
```
Comment (MichaelBoeser) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts]: correct description: ...is not displayed.

Comment (MichaelBoeser) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts]: Method is never used, is it required?

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts [hunk 3]
```diff
@@ -283,4 +301,48 @@ context('CPQ Configuration', () => {
...
+      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_ACC, 1);
+      configurationOverview.checkGroupHeaderDisplayed(GRP_CAM_IAW, 2);
+
+      type ovLineType = 'product' | 'simple';
+      const ovContent: { name: string; value: string; type: ovLineType }[] = [
+        { name: 'Camera Body', value: 'Nikon D850', type: 'product' },
+        { name: 'Memory Card', value: 'SanDisk Extreme Pro', type: 'product' },
+        { name: 'Lenses', value: 'Sigma 85mm F1.4 DG HS', type: 'product' },
+        { name: undefined, value: 'Nikon AF-P DX NIKKOR', type: 'product' },
+        // Display of Bag is a bug - https://github.com/SAP/spartacus/issues/10749
```
Comment (MichaelBoeser) [inline projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts]: should be fixed meanwhile, please remove this line

**Review by MichaelBoeser — CHANGES_REQUESTED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts (hunk 2 above)
Comment (Uli-Tiger) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts]: there was another bug(displaying empty groups), which is now merged and now also the method is called

**Review by MichaelBoeser — APPROVED** [review]

**Pull request merged**